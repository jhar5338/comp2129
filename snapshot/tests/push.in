push a 1
get a
set a 4 5
get a
push a 3 2 1
get a
del a
set a
get a
push a
get a
push b
bye
