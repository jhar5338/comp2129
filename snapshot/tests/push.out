> no such key

> no such key

> ok

> [4 5]

> ok

> [1 2 3 4 5]

> ok

> ok

> []

> ok

> []

> no such key

> bye
